birdhouse/components/weaver/post-docker-compose-up

```diff
@@ -246,7 +246,8 @@ for prov in ${WEAVER_WPS_PROVIDERS}; do
     if [ -z "${prov}" ]; then
         continue
     fi
-    prov_url="https://${PAVICS_FQDN_PUBLIC}${TWITCHER_PROTECTED_PATH}/${prov}"
+    prov_url="${prov#*:}"
+    prov="${prov%%:*}"
     prov_cap="${prov_url}?service=WPS&request=GetCapabilities"
```

Comment:
Just commenting here, but applies for all `default.env` of each WPS bird provider as well.

The public URL is intentionally employed such that permissions applied on those respective birds are enforced even when sending the execution request through Weaver. Using the internal URL bypasses this security check, which could allow a user to execute protected WPS using Weaver as a backdoor.

Reply:
I thought cowbird synced the permissions so that this wasn't necessary

Reply:
For the moment, Cowbird only syncs permissions of endpoints to access job outputs for the same user that pushed the job. 
The permissions between Weaver and the "remote" WPS is not synced.

Either way, I still prefer using the public URLs because they are displayed in `/weaver/providers` and `/weaver/providers/{name}` responses.